pkg/api/api.go

```diff
@@ -142,6 +143,8 @@ func (a API) ReceiveEvent(w http.ResponseWriter, r *http.Request) {
 
 	// If self hosting and keys are not defined, error.
 	if a.requireKeys && len(a.localEventKeys) == 0 {
+		a.log.Error().Msg("rejecting event; event keys are required to ingest events securely")
```

Comment:
Could we use a word instead of "ingest"? It reads confusingly given our company name